### PR TITLE
Tweak the hot reload update toast

### PR DESCRIPTION
**Pull request by pranavkm**
Animate

### CODE REVIEW

**Comment by pranavkm** [comment]
FYI @DamianEdwards

**Review by SteveSandersonMS — APPROVED** [review]

File: src/BuiltInTools/BrowserRefresh/WebSocketScriptInjection.js [hunk 1]
```diff
@@ -171,10 +171,10 @@ setTimeout(async function () {
...
+    el.setAttribute('style', 'z-index: 1000000; width: 48px; height: 48px; position:fixed; top:5px; left: 3px');
```
Comment (SteveSandersonMS) [inline src/BuiltInTools/BrowserRefresh/WebSocketScriptInjection.js:175]:
```suggestion
    el.setAttribute('style', 'z-index: 1000000; width: 48px; height: 48px; position:fixed; top:5px; left: 5px');
```

The asymmetry was bothering me.

**Comment by SteveSandersonMS** [comment]
@pranavkm FYI this change doesn't seem to be in `main`. Does it need to be forward-ported manually, or is there a difference process for the SDK repo?

**Comment by pranavkm** [comment]
Yeah, the SDK team merges forward from 6.0.1xx / rc2: https://github.com/dotnet/sdk/pull/21199.

**Pull request merged**